modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-raid_{context}"]
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume. {product-title} supports RAID 0, RAID 1, RAID 4, RAID 5, RAID 6, and RAID 10 for data protection and fault tolerance. See "About disk mirroring" for more details.
+
+. Create a Butane config that configures a data volume by using software RAID.
+
+* To configure a data volume with RAID 1 on the same disks that are used for a mirrored boot disk, create a `$HOME/clusterconfig/raid1-storage.bu` file, for example:
++
```

Comment:
For this file example and the later one, maybe add a ".Sample YAML configuration" or something along those lines.

Reply:
I wondered about that and opted to leave a header out because the lead-in sentence introduces the example. But based on your comment, I changed my mind and added to both examples...wouldn't hurt! Thanks!